app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -62,22 +63,23 @@
 
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/input_name"
-            android:fontFamily="@font/roboto_light_italic"
             android:layout_width="match_parent"
-            android:layout_height="50dp" />
+            android:layout_height="@dimen/signUp_50dp"
+            android:fontFamily="@font/roboto_light_italic"
+            tools:ignore="TextContrastCheck" />
     </com.google.android.material.textfield.TextInputLayout>
 
 
     <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/til_email"
         style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
-        android:layout_width="270dp"
-        android:layout_height="75dp"
+        android:layout_width="@dimen/signUp_270dp"
+        android:layout_height="@dimen/signUp_75dp"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks